### DISCUSSION TITLE
error with grpc_tools_node_protoc on Windows

### DISCUSSION THREAD

**Discussion by mizzet1** [deeplay-io/nice-grpc#533] [Q&A]
When I call grpc_tools_node_protoc in the terminal using WebStorm on Windows 11 I get this error: 
myprojectfolderpath/node_modules\grpc-tools\bin\protoc.js:41
    throw error;
    ^

I have a project with a single example.proto file and a compiled_proto folder and the command I'm calling from the terminal opened in my project folder is:
grpc_tools_node_protoc  --ts_proto_out=./compiled_proto  --ts_proto_opt="outputServices=nice-grpc,outputServices=generic-definitions,useExactTypes=false"  --proto_path=example.proto

**Comment by aikoven** [comment]
Can you please post the full output of the command?

**Reply by mizzet1** [reply]
C:\Users\Kuma\WebstormProjects\rpctest\node_modules\grpc-tools\bin\protoc.js:41
    throw error;
    ^

Error: Command failed: C:\Users\Kuma\WebstormProjects\rpctest\node_modules\grpc-tools\bin\protoc.exe --plugin=protoc-gen-grpc=C:\Users\Kuma\WebstormProjec
ts\rpctest\node_modules\grpc-tools\bin\grpc_node_plugin.exe --ts_proto_out=./compiled_proto --ts_proto_opt=outputServices=nice-grpc,outputServices=generic-definitions,useExactTypes=false --proto_path=example.proto

    at genericNodeError (node:internal/errors:984:15)
    at wrappedFn (node:internal/errors:538:14)
    at ChildProcess.exithandler (node:child_process:422:12)
    at ChildProcess.emit (node:events:519:28)
    at maybeClose (node:internal/child_process:1105:16)
    at Socket.<anonymous> (node:internal/child_process:457:11)
    at Socket.emit (node:events:519:28)
    at Pipe.<anonymous> (node:net:337:12) {
  code: 3221225781,
  killed: false,
  signal: null,
  cmd: 'C:\\Users\\Kuma\\WebstormProjects\\rpctest\\node_modules\\grpc-tools\\bin\\protoc.exe --plugin=protoc-gen-grpc=C:\\Users\\Kuma\\WebstormProjects\\
rpctest\\node_modules\\grpc-tools\\bin\\grpc_node_plugin.exe --ts_proto_out=./compiled_proto --ts_proto_opt=outputServices=nice-grpc,outputServices=generic-definitions,useExactTypes=false --proto_path=example.proto'
}

**Reply by aikoven** [reply]
A few solutions mentioning this error:

https://github.com/grpc/grpc-web/issues/697
https://stackoverflow.com/a/58440626

They mention that you need to install the VC Runtime.
